packages/manager/apps/pci-instances/public/translations/status/Messages_fr_FR.json

```diff
@@ -0,0 +1,38 @@
+{
+  "ACTIVE": "Activée",
```

Comment:
Translation doesn't seem correct, if you are using past participle in french it should ACTIVATED in the english key